exchanges/huobi/huobi_websocket.go

```diff
@@ -125,7 +127,6 @@ func (h *HUOBI) wsFunnelConnectionData(ws stream.Connection, url string) {
 
 // wsReadData receives and passes on websocket messages for processing
 func (h *HUOBI) wsReadData() {
-	h.Websocket.Wg.Add(1)
 	defer h.Websocket.Wg.Done()
```

Comment:
This function has no way of exiting and will cause a wait. Please add a shutdown listener or pass the `resp` to `comms` in `wsFunnelConnectionData` when `resp.Raw == nil` before it returns. A shutdown listener sounds simpler though